Permanent/ViewControllers/MainViewController.swift

```diff
@@ -13,10 +13,14 @@ import WebKit
 class MainViewController: BaseViewController<MyFilesViewModel> {
     @IBOutlet var directoryLabel: UILabel!
     @IBOutlet var backButton: UIButton!
-    @IBOutlet var tableView: UITableView!
+    @IBOutlet weak var collectionView: UICollectionView!
+//    @IBOutlet var tableView: UITableView!
```

Comment:
Maybe some code comments in this file should be deleted, now that grid view works.